### ISSUE TITLE
I wonder what to solve "Cannot interpret feed_dict key as Tensor is not an element of this graph." problem

### ISSUE THREAD

**Issue by dx-han** [jaungiers/Multidimensional-LSTM-BitCoin-Time-Series#5]
When I predict the result, the kernel told me this warning, so I have no idea what's going wrong.

**Comment by johndpope** [comment]
try this
https://twitter.com/PeterLBrandt/status/984776999393808385

at end of day - you're going to see conflicting indicators to buy or sell depending on time frame.
https://www.tradingview.com/symbols/BTCUSD/technicals/

the volume spike is important to driving trend.

**Comment by SanderRednas** [comment]
Yes well instead of going to twitter I found the problem to be in the threading.Thread call, instead I generate the model directly as such and it worked:

`# Generate the model
model = lstm_bot.lstm.build_network([ncols, 150, 150, 1])
model.fit_generator(
    data_gen_train,
    steps_per_epoch=steps_per_epoch,
    epochs=configs['model']['epochs']
)
model.save(configs['model']['filename_model'])
print('> Model Trained! Weights saved in', configs['model']['filename_model'])

`
